### PR TITLE
Fix build command

### LINKED ISSUE #158: wrong go env in Makefile?
https://github.com/kubernetes/cloud-provider-aws/blob/1be8de7a3cc579c8ab2619d70b0769cbfd5371da/Makefile#L48

should be `GO111MODULE` & `CGO_ENABLED`?

### PR DESCRIPTION
**Pull request by ayberk**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
Fixes the env vars for the build command.

**Which issue(s) this PR fixes**:
Fixes #158

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

**Review by nckturner — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -45,7 +45,7 @@ RELEASE_TAG := $(shell git describe --abbrev=0 2>/dev/null)
 RELEASE_IMAGE ?= $(REGISTRY)/$(IMAGE_NAME):$(RELEASE_TAG)
 
 aws-cloud-controller-manager: $(SOURCES)
-	 CGO111MODULE=on GO_ENABLED=0 GOOS=$(GOOS) go build \
+	 GO111MODULE=on CGO_ENABLED=0 GOOS=$(GOOS) go build \
```
Comment (andrewsykim) [inline Makefile:48]: funny typo

**Review by andrewsykim — COMMENTED** [review]
/ok-to-test

File: Makefile (hunk 1 above)
Comment (nckturner) [reply Makefile:48]: _The mystery of the lost "C"_

**Pull request merged**